Gems/LyShine/Code/Source/Draw2d.cpp

```diff
@@ -103,6 +103,11 @@ void CDraw2d::OnBootstrapSceneReady(AZ::RPI::Scene* bootstrapScene)
     AZ::RPI::SceneId sceneId = scene->GetId();
     LyShinePassRequestBus::EventResult(uiCanvasPass, sceneId, &LyShinePassRequestBus::Events::GetUiCanvasPass);
 
+    if (!uiCanvasPass) {
+        AZ_Warning("Draw2d", false, "No UiCanvasPass found in Render-Pipeline!");
```

Comment:
I'm wondering if that is something that we should recover from, the lack of scene above is asserting for example. In any case an `AZ_Error` sounds more suited to the severity. What do you think?
Super minor but we tend to keep assert messages more neutral, without exclamation marks (that applies to the previous message as well)

Reply:
done.

Looking over this again i am somewhat worried tough that the m_dynamicDraw - member stays uninitialized there though, and the only reason this doesn't crash is because the UiRenderer realizes independently that it's DynamicDrawContext fails to initialize, and therefore doesn't call the various render methods of this class.

Since the uiCanvasPass there only provides the drawlistTag, would it be a good idea to just not to call the InitDrawListTag of the DynamicDrawContext, and leave it existing, but in a Non-Ready state?